### ISSUE TITLE
Decimal values in @keyframe definitions not highlighted properly

### ISSUE THREAD

**Issue by martinAnsty** [y0ssar1an/CSS3#22]
In an `@keyframes` definition values including decimals (eg. `33.33% { }`) are not highlighted properly. Only the figures after the decimal point are highlighted. Presumably the regex for this is only matching numbers, just needs periods adding.

**Comment by pi241a** [comment]
It seems like ``@keyframes`` highlighting is kind of spotty in general.

````
@keyframes ident {
	0% {
		left: 0;
		top: 0;
	}

	30.1% {
		top: 50px;
	}

	68%, 
	72% {
		left: 50px;
	}

	100% {
		left: 100%;
		top: 100px;
	}
}
````

![keyframes](https://cloud.githubusercontent.com/assets/4296393/6910603/21d7d082-d70a-11e4-8c6f-0ecd57627a8c.png)

**Comment by y0ssar1an** [comment]
fixed in v1.2.12

@martinAnsty Thanks very much for taking the time to report this
